tools/blocksconvert/scanner/scanner.go

```diff
@@ -92,8 +95,13 @@ func NewScanner(cfg Config, scfg blocksconvert.SharedConfig, l log.Logger, reg p
 		}
 	}
 
+	var users = blocksconvert.AllowAllUsers
+	if cfg.AllowedUsers != "" {
+		users = blocksconvert.ParseAllowedUsers(cfg.AllowedUsers)
+	}
+
 	var ignoredUserRegex *regexp.Regexp = nil
-	if cfg.IgnoredUserPattern != "" {
+	if users.AllUsersAllowed() && cfg.IgnoredUserPattern != "" {
```

Comment:
`!users.AllUsersAllowed()`?

Reply:
This is correct. If all users are allowed (no config option for allowed users was passed), only then we want to ignore some users.

If "allowed users" option was used, only plans for those specific users are generated.

Reply:
Right, I confused `Only used if all -scanner.allowed-users is not set.` to be allow all users :)

Reply:
Perhaps I should improve description strings?